tests/python/pants_test/backend/python/tasks/native/test_ctypes.py

```diff
@@ -55,15 +56,15 @@ def run_before_task_types(cls):
       'dependencies': ['src/python/plat_specific_c_dist:ctypes_c_library'],
       'filemap': {
         '__init__.py': '',
-        'setup.py': """\
-from setuptools import setup, find_packages
-setup(
-  name='platform_specific_ctypes_c_dist',
-  version='0.0.0',
-  packages=find_packages(),
-  data_files=[('', ['libc-math-lib.so'])],
-)
-        """,
+        'setup.py': dedent("""\
```

Comment:
Bless you for adding dedent :D

Reply:
In the future, if you can do this in a separate PR (which is nicely trivial to review), it would help to reduce the noise in this review (and adds the benefit that reverts and merges can be better targeted) :)